cellxgene_schema_cli/cellxgene_schema/schema_definitions/schema_definition.yaml

```diff
@@ -316,9 +316,28 @@ components:
                     - MmusDv:0000001
               exceptions:
                 - unknown
-        # If organism is not humnan nor mouse
+          - # If organism is Zebrafish
+            rule: "organism_ontology_term_id == 'NCBITaxon:7955'"
+            error_message_suffix: >-
+              When 'organism_ontology_term_id' is 'NCBITaxon:7955' (Danio rerio),
+              'development_stage_ontology_term_id' MUST be the most accurate descendant of 'ZFS:0000001' and it
+              MUST NOT be 'ZFS:0000000' for Unknown. The str 'unknown' is acceptable.
+            type: curie
+            curie_constraints:
+              ontologies:
+                - ZFS
+              allowed:
+                ancestors:
```

Comment:
It is a little confusing, but we use the ['allowed']['ancestors'] key to mean: we only allow terms that have one of the following terms as an ancestor

(NOT we allow ancestors of the one of the following terms)